tests/Feature/MeetingTest.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Meeting;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MeetingTest extends TestCase
+{
+    use RefreshDatabase, WithFaker;
+
+    public function test_create()
+    {
+        $organizer = factory(User::class)->create();
+
+        $startTime = $this->faker->dateTimeBetween('now', '+2 years');
+        $endTime = $this->faker->dateTimeBetween(
+            $startTime->format('Y-m-d H:i:s').' +1 hours',
+            $startTime->format('Y-m-d H:i:s').' +6 hours'
+        );
+
+        $response = $this->actingAs($organizer, 'api')->postJson(
+            route('api.meeting.create'),
+            [
+                'name' => $this->faker->company,
+                'description' => $this->faker->paragraph,
+                'location' => $this->faker->address,
```

Comment:
I like putting actual values here instead of faker data. I've found that faker data can sometimes cause intermittent failures with weird characters.